cmd/gaia/app/sim_test.go

```diff
@@ -55,7 +59,34 @@ func init() {
 	flag.IntVar(&period, "SimulationPeriod", 1, "Run slow invariants only once every period assertions")
 }
 
-func appStateFn(r *rand.Rand, accs []simulation.Account, genesisTimestamp time.Time) json.RawMessage {
+func appStateFromGenesisFileFn(r *rand.Rand, accs []simulation.Account, genesisTimestamp time.Time) (json.RawMessage, []simulation.Account, string) {
+	var genesis tmtypes.GenesisDoc
+	cdc := MakeCodec()
+	bytes, err := ioutil.ReadFile(genesisFile)
+	if err != nil {
+		panic(err)
+	}
+	if err = cdc.UnmarshalJSON(bytes, &genesis); err != nil {
+		panic(err)
+	}
+	var appState GenesisState
+	if err = cdc.UnmarshalJSON(genesis.AppState, &appState); err != nil {
+		panic(err)
+	}
+	var newAccs []simulation.Account
+	for _, acc := range appState.Accounts {
+		// Pick a random private key, since we don't know the actual key
+		// This should be fine as it's only used for mock Tendermint validators
```

Comment:
We should probably mention in this comment that these key are never verified by mock tendermint

Reply:
so mentioned